Makefile

```diff
@@ -18,56 +16,58 @@ all: help
 
 .PHONY: help
 help:
-	@cat README.md
-
-.PHONY: run
-run: bin/python setup.py
-	PYTHONPATH=src bash -c 'bin/python src/metaswitch/crest/main.py'
+	@cat docs/development.md
 
 .PHONY: test
-test: bin/python setup.py
-	bin/python setup.py test
+test: setup.py env
+	PYTHONPATH=src:common ${ENV_DIR}/bin/python setup.py test -v
+
+${ENV_DIR}/bin/flake8: env
```

Comment:
According to @mbt-msw you shouldn't have a real thing depend on a `.PHONY` thing (as `.PHONY` messes with error detection in weird and unexpected ways).  Instead, depend on the sentinel file.